docs/README_RECON.md

```diff
@@ -138,30 +212,79 @@ class ColumnMapping:
 
 ### transformations
 
-```python
+<table>
+<tr>
+<th>python</th>
+<th>json</th>
+</tr>
+<tr>
+<td>
+<pre lang="python">
+
 @dataclass
 class Transformation:
     column_name: str
     source: str
     target: str | None = None
-```
+
+</pre>
+</td>
+<td>
+<pre lang="json">
+"transformations":[
+    {
+      "column_name": "product_price",
```

Comment:
Please update all the actual values with <TO-BE-FILLED> elements.

Reply:
Fixed